platform-core-module/src/main/java/com/softicar/platform/core/module/file/stored/AGStoredFile.java

```diff
@@ -210,6 +210,19 @@ public boolean hasMimeTypeOrExtension(MimeType mimeType) {
 		return hasMimeType(mimeType) || mimeType.getFilenameSuffixes().stream().anyMatch(this::hasFileNameExtension);
 	}
 
+	/**
+	 * Sets the given {@link IMimeType} as {@link AGStoredFile#CONTENT_TYPE}.
+	 *
+	 * @param mimeType
+	 *            the {@link IMimeType} (never <i>null</i>)
+	 * @return this
+	 */
+	public AGStoredFile setContentTypeByMimeType(IMimeType mimeType) {
```

Comment:
why not call it just `setContentType`, otherwise its existence may be overlooked

Reply:
ok